pkg/pillar/cmd/diag/diag.go

```diff
@@ -981,32 +978,27 @@ func parsePrint(configURL string, resp *http.Response, contents []byte) {
 		return
 	}
 
-	configResponse, err := readConfigResponseProtoMessage(contents)
+	uuidResponse, err := readUUIDResponseProtoMessage(contents)
```

Comment:
Can you also rename the configURL parameter above to reqURL or uuidURL? (Yetus might argue the latter should have capital UUID).

Reply:
Fixed.